contrib/guix/manifest.scm

```diff
@@ -123,6 +123,104 @@ desirable for building Bitcoin Core release binaries."
   (package-with-extra-patches mingw-w64-x86_64-winpthreads
     (search-our-patches "winpthreads-remap-guix-store.patch")))
 
+
+;; While mingw-w64 is packaged in Guix, we maintain our own package,
+;; to use UCRT runtime.
+(define* (make-mingw-w64/implementation machine
```

Comment:
> we maintain our own package

If we are going to copy-paste this from Guix, and maintain it outself, seems like we should simplify it for our usage?  i.e we know we are building winpthreads.

Reply:
Thanks! [Reworked](https://github.com/bitcoin/bitcoin/pull/33593#issuecomment-3488313837).